batch/batch/driver/create_instance.py

```diff
@@ -249,21 +262,26 @@ async def create_instance(app, zone, machine_name, machine_type, activation_toke
 -e BATCH_LOGS_BUCKET_NAME=$BATCH_LOGS_BUCKET_NAME \
 -e INSTANCE_ID=$INSTANCE_ID \
 -e PROJECT=$PROJECT \
+-e ZONE=$ZONE \
 -e WORKER_CONFIG=$WORKER_CONFIG \
 -e MAX_IDLE_TIME_MSECS=$MAX_IDLE_TIME_MSECS \
 -e WORKER_DATA_DISK_MOUNT=/mnt/disks/$WORKER_DATA_DISK_NAME \
 -e BATCH_WORKER_IMAGE=$BATCH_WORKER_IMAGE \
+-e UNRESERVED_WORKER_DATA_DISK_SIZE_GB=$UNRESERVED_WORKER_DATA_DISK_SIZE_GB \
 -v /var/run/docker.sock:/var/run/docker.sock \
 -v /usr/bin/docker:/usr/bin/docker \
 -v /usr/sbin/xfs_quota:/usr/sbin/xfs_quota \
--v /batch:/batch \
+-v /batch:/batch:shared \
 -v /logs:/logs \
 -v /gcsfuse:/gcsfuse:shared \
 -v /xfsquota:/xfsquota \
 --mount type=bind,source=/mnt/disks/$WORKER_DATA_DISK_NAME,target=/host \
+--mount type=bind,source=/dev,target=/dev,bind-propagation=rshared \
 -p 5000:5000 \
 --device /dev/fuse \
 --device $XFS_DEVICE \
+--device /dev \
+--privileged \
```

Comment:
Are we sure privileged is still necessary?

Reply:
I'll try again, but I think so. There must be other things privileged does that are not capabilities. I think it might be something with the namespace.